### ISSUE TITLE
pip install fails on MacOS Sierra

### ISSUE THREAD

**Issue by wlbentley** [eendebakpt/oapackage#9]
I'm trying to install OSpackage on MacOS Sierra:
```
pip install OApackage
```
- XCode 8.3.2
- Python 3.6
- Pip 9.0.1
- SWIG Version 3.0.12
- numpy 1.12.1

The install fails with clang errors as in this [Gist](https://gist.github.com/wlbentley/f4ee385cc0f81de15c234a2ea8f7c3df).

Any thoughts on how to get this install to work?

Thanks.

**Comment by peendebak** [comment]
@wlbentley This was fixed in #7. Can you try
```
pip install git+https://github.com/eendebakpt/oapackage.git
```

**Comment by wlbentley** [comment]
Perfect. Thanks so much.
